test/TestExtensions/DefaultClusterFixture.cs

```diff
@@ -8,8 +8,11 @@ public class DefaultClusterFixture : BaseTestClusterFixture
         protected override TestCluster CreateTestCluster()
         {
             var options = new TestClusterOptions();
+#if !NETSTANDARD_TODO
+            // waiting for OrleansProviders project to be ported over to vNext
             options.ClusterConfiguration.AddMemoryStorageProvider("Default");
             options.ClusterConfiguration.AddMemoryStorageProvider("MemoryStore");
+#endif
```

Comment:
heh, actually git was smart about it :)
Anyway, pushed a rebased version now that your PR was merged.
Also included the missing collection fixture, otherwise tests that used `HostedTestClusterEnsureDefaultStarted` were not starting the silo cluster